Mathlib/Topology/Support.lean

```diff
@@ -399,3 +401,10 @@ theorem exists_finset_nhd_mulSupport_subset {f : ι → X → R}
 #align locally_finite.exists_finset_nhd_support_subset LocallyFinite.exists_finset_nhd_support_subset
 
 end LocallyFinite
+
+@[to_additive]
+theorem _root_.locallyFinite_mulSupport_iff [CommMonoid M] {f : ι → X → M} :
```

Comment:
no need for the _root_?

Reply:
Indeed, an oversight from shuffling things around. Fixed.